index.html

```diff
@@ -0,0 +1,182 @@
+
+  <main>
+    <section id="hero-container">
+        <div>
+            <h2 class="fs-secondary-heading fw-bold text-accent-400 ">"Welcome to Div News"</h2>
+            <div class="hero-headline-title">  
+            <h1 class="hero-headline-title text-accent-400 ">Your number one source <br>for Tech related news</h1>
+          </div>
+          
+          <div class="blog-updates">
+            <p class="fs-600">Stay up to date with the latest in tech simply, by bookmarking our page to get notifications and <br> stay in the loop on all the tech trends.</p>
+          </div>
+            <h2 class="fs-secondary-heading text-neutral-100 fw-bold ">Updated: Mon - Fri</h2>
+            <h3 class="fs-600 text-neutral-900">bookmark us and stay in the loop</h2>
+          </div>
+        </div>
+      </div>
+    </section>
+
+      <!-- hero section ends here -->
+
+      <!-- tech topics section starts here -->
+
+    <section id="tech-topics">
+      <div class="container">
+        <div>
+          <h2 class="title fs-700 text-accent-100 fw-semi-bold" >Tech Topics</h2>
+          <div class="indicator"></div>
+        </div>
+        <div class="tech-topics-container">
+          <div class="tech-topics-card">
+            <img class="tech-logo" src="img/html-5.png" alt="Html-logo">
+            <h3 class="fw-bold fs-500">HTML</h3>
+            <p class="tech-topic-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="tech-topics-card">
+            <img class="tech-logo"src="img/css-3.png" alt="Css-logo">
+            <h3 class="fw-bold fs-500">CSS</h3>
+            <p class="tech-topic-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="tech-topics-card">
+            <img class="tech-logo" src="img/bootstrap.png" alt="bootstrap-logo">
+            <h3 class="fw-bold fs-500">Bootstrap</h3>
+            <p class="tech-topic-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="tech-topics-card">
+            <img class="tech-logo" src="img/js.png" alt="js-logo">
+            <h3 class="fw-bold fs-500">JavaScript</h3>
+            <p class="tech-topic-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
+          <div class="tech-topics-card">
+            <img  class="tech-logo" src="img/java.png" alt="java-logo">
+            <h3 class="fw-bold fs-500">Java</h3>
+            <p class="tech-topic-desc">Lorem ipsum dolor, sit amet consectetur adipisicing elit.</p>
+          </div>
+        </div>
+          <a href="#" class="see-more">Check Out Our Dev Topics</a>
+          <div class="btn-wrap">
+          <button class="see-more-btn">Dev Topics</button>
+          </div>
+      </div>
+    </section>
+    
+      <!-- tech topics section ends here -->
+
+      <!-- contributors section starts here -->
+
+    <section id="contributors">
```

Comment:
- [ ] I would like to recommend you work on all your section and page mobile design. For example the contributor section, there are a few differences between your design and the original one. Please fix the design for all your mobile version for all sections and page:+1: 

## Example of contributor section  

#### Figma design
![Screenshot from 2022-10-28 19-24-20](https://user-images.githubusercontent.com/6615394/198717974-f9843ebd-01a3-4a8d-8c77-a68210743648.jpg)

#### Original design
![Screenshot from 2022-10-28 19-20-02](https://user-images.githubusercontent.com/6615394/198717981-8a498b7d-e4f0-4f2c-8146-1f6b20f12100.jpg)